repobee_sanitizer/_syntaxhelpers.py

```diff
@@ -0,0 +1,112 @@
+"""Functions and constants relating to the syntax of repobee-sanitizer.
+
+.. module:: _syntaxhelpers
+    :synopsis: Functions and constants relating to the syntax of
+    repobee-sanitizer.
```

Comment:
This line must be indented to count as part of the `synopsis` directive.
```suggestion
        repobee-sanitizer.
```